src/drc/datamodel/tests/factories.py

```diff
@@ -18,6 +18,10 @@ class EnkelvoudigInformatieObjectFactory(factory.django.DjangoModelFactory):
     formaat = 'some formaat'
     taal = 'dut'
     inhoud = factory.django.FileField(data=b'some data', filename='file.bin')
+    link = 'http://www.example.com/'
+    beschrijving = 'some beschrijving'
+    vertrouwelijkaanduiding = VertrouwelijkheidsAanduiding.openbaar
+    informatieobjecttype = 'https://example.com/ztc/api/v1/catalogus/1/informatieobjecttype/1'
```

Comment:
`factory.Faker('url')` is misschien fijner om wat randomness te hebben hierin?

Reply:
Kan, maar de rest is ook statisch?

Reply:
persoonlijke voorkeur gaat zelfs uit naar het niet declareren omdat het geen verplicht veld is.

En als de rest behouden blijft, zou ik deze ook niet statisch maken (zoals `factory.Faker('paragraph')` bijvoorbeeld voor `beschrijving`.